maquetado/scss/styles.scss

```diff
@@ -107,3 +113,25 @@
 .height {
   line-height: 24px;
 }
+
+.navbar {
+  background-color: $white;
+  box-shadow: 0 4px 10px 0 $black-05;
+  height: 70px;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  &::before {
+    content:'';
+    border-top: 4px solid $cerulean;
+    height: 4px;
+    position: absolute;
+    width: 100%;
+  }
+}
+
+.navbar-logo {
+  height: 70px;
+  width: 210px;
+  margin-left: 220px;
```

Comment:
Remember the alphabetical order.Also, you can use padding to set inner elements, in this case the logo's position. We usually use margin for separating several siblings from each other, but when it comes to setting a certain distance between a parent tag and its child you should always use padding.